legal/privacy-policy.md

```diff
@@ -0,0 +1,268 @@
+
+To find out more information about Cookies generally, including information about how to manage and delete Cookies, please visit [https://allaboutcookies.org/](https://allaboutcookies.org/) or [https://ico.org.uk/for-the-public/online/cookies/](https://ico.org.uk/for-the-public/online/cookies/) if you are located in the European Union.
+
+## Data Security
+
+We endeavor to protect your Personal Data from unauthorized access, use and disclosure using appropriate physical, technical, organizational and administrative security measures based on our Services,the type of Personal Data being collected and how we are processing that data. You should also help protect your data by selecting and protecting your password and/or other sign-on mechanism(s) with care; limiting access to your computer or device and browser; and signing off after you have finished accessing your account. Although we work to protect the security of your account and other data that we hold in our records, be aware that no method of transmitting data over the internet or storing data is completely secure.
+
+## Data Retention
+
+We retain Personal Data about you for as long as reasonably necessary to provide you with our Services or otherwise in support of our business or commercial purposes for utilization of your Personal Data, as expressed. When establishing a retention period for particular categories of data, we consider who we collected the data from, our need for the Personal Data, why we collected the Personal Data, and the sensitivity of the Personal Data. In some cases we retain Personal Data for a longer period, if doing so is necessary to comply with our legal obligations, resolve disputes or collect fees owed, or as is otherwise permitted or required by applicable law, rule or regulation. We may further retain information in an anonymous or aggregated form where such information would not identify you personally.
+
+For example:
+
+- We retain your profile information and credentials for as long as you have an account with us.
+- We retain your payment data for as long as we need to process your purchase or subscription.
+- We retain your device/IP data for as long as we need it to ensure that our systems are working appropriately, effectively and efficiently.
+
+It's worth noting that we avoid retaining data unless necessary to provide our Service. For example:
+
+- We do not currently store source code that we proxy during collaboration sessions.
+- We do not currently retain audio or video recordings of Collaboration calls after a call is completed.
```

Comment:
We do not retain audio or video recordings of Collaboration calls. All media transmission is handled via Livekit. 

Or something like this...